app/assets/javascripts/templates/management/wysiwyg-sidebar.hbs

```diff
@@ -11,4 +11,8 @@
     <svg width="19" height="14" viewBox="0 0 19 14" xmlns="http://www.w3.org/2000/svg"><path d="M2.375 11.667l4.75-8.391 4.75 5.656 2.375-2.21 2.375 4.945H2.375zm11.281-9.334c.985 0 1.781.784 1.781 1.75s-.796 1.75-1.78 1.75c-.985 0-1.782-.784-1.782-1.75s.797-1.75 1.781-1.75zM0 14h19V0H0v14z" fill="#FFF" fill-rule="evenodd"/></svg>
     <span>Image</span>
   </button>
+  <button type="button" class="js-add-html">
+    <svg width="23px" height="21px" viewBox="0 0 23 21" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"> <g id="CManager" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"> <g id="CM-Page-a4" transform="translate(-739.000000, -1342.000000)" fill="#FFFFFF"> <g id="Group-3" transform="translate(247.000000, 1330.000000)"> <g id="Group-2" transform="translate(221.000000, 0.000000)"> <g id="Group-12-Copy" transform="translate(260.000000, 0.000000)"> <g id="Group-13" transform="translate(11.000000, 12.000000)"> <path d="M20.1376978,10.635465 L15.9380384,14.8351245 L17.1698655,16.0669516 L21.3695249,11.8672921 L21.4056218,11.903389 L22.6374488,10.6715619 L22.601352,10.635465 L22.6374488,10.5993682 L21.4056218,9.36754111 L21.3695249,9.40363795 L17.1698655,5.20397851 L15.9380384,6.43580559 L20.1376978,10.635465 Z M2.827263,10.3303268 L7.02692245,6.13066732 L5.79509536,4.89884024 L1.59543592,9.09849968 L1.55933908,9.06240284 L0.327511996,10.2942299 L0.363608836,10.3303268 L0.327511996,10.3664236 L1.55933908,11.5982507 L1.59543592,11.5621538 L5.79509536,15.7618133 L7.02692245,14.5299862 L2.827263,10.3303268 Z M13.9729104,0.162342844 L15.8656292,0.808574757 L9.02708958,20.8376572 L7.13437082,20.1914252 L13.9729104,0.162342844 Z" id="-icon"></path> </g> </g> </g> </g> </g> </g> </svg>
```

Comment:
Did you optimise this SVG (with a tool like [svgo](https://github.com/svg/svgo))?